content/en/users/fedcloud/solutions/ecas.md

```diff
@@ -0,0 +1,254 @@
+    powered on on-demand.
+
+![Cluster size and name.](../ecas-size.png)
+
+-   **Resume and Launch**: a summary of the chosen cluster
+    configuration. To start the deployment process, click the Submit
+    button.
+
+![Resume and Launch.](../ecas-summary.png)
+
+When the front-end node of the cluster has been successfully deployed,
+the user will be notified with the credentials to access via SSH.
+
+![ECAS cluster connection details.](../ecas-end.png)
+
+The cluster details are available by clicking on the \"Manage your
+deployed clusters\" link on the front page:
+
+![Manage your clusters.](../ecas-manage.png)
+
+::: {.note}
+::: {.title}
+Note
+:::
+
+The configuration of the cluster may take some time. Please wait for its
+completion before starting to start using the cluster.
+:::
+
+### Accessing the cluster
+
+To access the front-end of the elastics cluster:
+
+-   download the SSH private key provided by the EC3 portal;
+-   change its permissions to `600`;
+-   access via SSH providing the key as identity file for public key
+    authentication.
+
+``` {.console}
+[fabrizio@MBP EC3]$ ssh -i key.pem cloudadm@134.158.151.218
+Last login: Mon Nov 18 11:37:29 2019 from torito.i3m.upv.es
+[cloudadm@oph-server ~]$ sudo su -
+[root@oph-server ~]#
+```
+
+Both the front-end and the working node are configured by Ansible. This
+process usually takes some time. User can monitor the status of the
+cluster configuration using the `is_cluster_ready` command-line tool:
+
+``` {.console}
+[root@oph-server ~]# is_cluster_ready
+Cluster is still configuring.
+```
+
+The cluster is successfully configured when the command returns the
+following message:
+
+``` {.console}
+[root@oph-server ~]# is_cluster_ready
+Cluster configured!
+```
+
+As SLURM is used as workload manager, it is possible to check the status
+of the working nodes by using the sinfo command, which provides
+information about Slurm nodes and partitions.
+
+``` {.console}
+[root@oph-server ~]# sinfo
+PARTITION AVAIL  TIMELIMIT  NODES  STATE NODELIST
+debug*       up   infinite   1  down* oph-io2
+debug*       up   infinite   1   idle oph-io1
+```
+
+### Accessing the scientific eco-system
+
+ECASLab provides two different ways to get access to its scientific
+eco-system: Ophidia client (`oph_term`) and JupyterHub.
+
+#### Perform some basic operations with Ophidia
+
+Run the Ophidia terminal as `ophuser` user.
+
+![Ophidia terminal.](../ecas-oph_term.png)
+
+The default parameters are already defined as environmental variables
+inside the `.bashrc` file:
+
+``` {.console}
+export OPH_SERVER_HOST="127.0.0.1"
+export OPH_SERVER_PORT="11732"
+export OPH_PASSWD="abcd"
+export OPH_USER="oph-test"
+```
+
+Create an empty container and a new datacube with random data and
+dimensions.
+
+![Create container (1).](../ecas-container-1.png)
+
+![Create container (2).](../ecas-container-2.png)
+
+Now, you can submit your first operation of data transformation: let's
```

Comment:
Before the users where refered as `the user` and here it starts to use `you`, it should be harmonized